controller_manager/src/controller_manager.cpp

```diff
@@ -379,14 +379,15 @@ controller_interface::return_type ControllerManager::switch_controller(
           if (strictness == controller_manager_msgs::srv::SwitchController::Request::STRICT) {
             RCLCPP_ERROR(
               get_logger(),
-              "Could not %s controller with name '%s' because no controller with this name exists",
+              "Could not '%s' controller with name '%s' because " +
+              "no controller with this name exists",
```

Comment:
```suggestion
              std::string("Could not '%s' controller with name '%s' because ") +
              std::string("no controller with this name exists"),
```

Reply:
Isn't this a bit verbose? The compiler will emit equivalent code with and without the explicit ctor there, and the pre-processor will concatenate strings which span multiple lines already.

I'd suggesting making use of that functionality: it's both more readable and less code.